src/Microsoft.Health.Encryption/Microsoft.Health.Encryption.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Encryption utilities, including those for customer-managed keys, used by Microsoft Health.</Description>
+    <TargetFrameworks>$(SupportedFrameworks)</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Security.KeyVault.Keys" />
+    <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks" />
```

Comment:
Double-checking again, is this really necessary?

Reply:
Removed from Encryption project